docs/documentation/Realtime/recent additions.md

```diff
@@ -2,22 +2,7 @@
 
 The GTFS Realtime Specification is not set in stone. Instead, it is an open specification developed and maintained by the community of transit agencies, developers, and other stakeholders who use GTFS Realtime. It is expected that this community of producers and consumers of GTFS Realtime data will have proposals for extending the spec to enable new capabilities.
 
-To contribute to GTFS Realtime, read the [Specification Amendment Process](../process) and follow the discussions in the open <a href="https://github.com/google/transit/issues" target="_blank">issues</a> and <a href="https://github.com/google/transit/pulls" target="_blank">pull requests</a> on Google's Transit Github repository (<a href="https://github.com/google/transit" target="_blank">google/transit</a>). ![](../../assets/mark-github.svg)
-
-
-!!! note ""
-
-	The official specification, reference and documentation are written in English. If a translation to a different language differs from the English original, the latter takes precedence. All communication is performed in English.
-
-<!-- <br><div class="landing-page">
-    <a class="button" href="../process">Specification Amendment Process</a><a class="button" href="../guiding-principles">Guiding Principles</a><a class="button" href="../revision-history">Revision History</a><a class="button" href="../extensions">Realtime Extensions</a>
-</div> -->
-
-## Active Proposals &ensp;<img src="../../assets/pr-active.svg" style="height:1em;"/>
-
-<!-- Active proposals for new features in GTFS Realtime.  -->
-
-Join the discussions on <a href="https://github.com/google/transit/pulls" target="_blank">Github</a> !
+To contribute to GTFS Realtime, read the [Specification Amendment Process](../reference) and follow the discussions in the open <a href="https://github.com/google/transit/issues" target="_blank">issues</a> and <a href="https://github.com/google/transit/pulls" target="_blank">pull requests</a> on Google's Transit Github repository (<a href="https://github.com/google/transit" target="_blank">google/transit</a>). ![](../../assets/mark-github.svg)
```

Comment:
> (../reference)

Should this go to the _Community_ page instead?

> read the [Specification Amendment Process]

Nit: I think we should always call them "GTFS Schedule Amendment Process" and "GTFS Realtime Amendment Process"; for a long time I thought they had the same amendment process because we call them both "Spec Amendment Process".
It's also how they are called on the Community page:
<img width="168" alt="Screenshot 2024-05-27 at 11 14 23 AM" src="https://github.com/MobilityData/elrond.gtfs.org/assets/63653518/6277b8a5-eb41-4426-b270-ba24f5145e4e">

> on Google's Transit Github repository

Maybe just _to the GTFS GitHub repository_, to promote that GTFS is an open standard, and the G has moved from _Google_ to _General_.